### PR TITLE
Milestone Three: Logic

### PR DESCRIPTION
**Pull request by Zubenna**
## In this milestone, I did the following;
- Created a JavaScript module containing the function calculate.js.
- Calculate.js takes in calculator object and button name as an argument to implement a part of the calculator logic.
- Created another module that contains the function operate.js.
- Operate.js implements aspects of the logic using the operation button name supplied to it.
- Calculate function is imported into App.js.

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  } else if (operators.includes(buttonName)) {
+    if (buttonName === '%') {
+      if (!next) {
+        next = total;
+      }
+      operation = buttonName;
+      total = operate(null, next, operation);
+      next = null;
+      operation = null;
```
Comment (Torres-ssf) [inline src/logic/calculate.js:24]: - [ ] The modifier operate should turn the number into a percentage based number, e.g. `95 % == .95`. We also need to move it to the operate module.
Screenshot from 2021-03-11 09-22-16

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @Zubenna :hand: 
Awesome job on this milestone so far. But we need to make some changes before moving on.

- [ ] No description was added to the PR. We need to edit the PR description and summarize all changes made in this PR. We can check this link [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) to learn more about how you can improve the professionalism of your repository.

Happy coding :computer: 
**Sérgio Torres**

File: src/logic/calculate.js (hunk 1 above)
Comment (Zubenna) [reply src/logic/calculate.js:24]: Done

**Review by Forison — APPROVED** [review]
# Status::Approved✔️

#### Good job on adding a descriptive pull request, kindly go ahead and merge the branch and proceed to your next task💪.

#### Happy coding👨‍💻

Addo Forison

**Pull request merged**